### ISSUE TITLE
 "rewardsScheduledForAddress": "0.0"

### ISSUE THREAD

**Issue by ricarper** [filecoin-station/core#469]
The gateways have just started (at this time) showing “rewardsScheduledForAddress”: “0.0” and there have been no payments to the address. Are there problems with the stations?

**Comment by ricarper** [comment]
It has recovered the accumulated now, but sometimes it shows 0 again and then recovers. Is everything OK with the stations?

**Comment by juliangruber** [comment]
It shows "0" when there's an error fetching the rewards. Would it be more helpful to show an error message instead?

When there's an error fetching rewards it's an issue with the JSON-RPC nodes we they use, not the stations themselves

**Comment by ricarper** [comment]
> It shows "0" when there's an error fetching the rewards. Would it be more helpful to show an error message instead?
> 
> When there's an error fetching rewards it's an issue with the JSON-RPC nodes we they use, not the stations themselves

No error is returned. It's when the container restarts, I checked that a few hours ago. At startup it starts at 0 and not the scheduled one. It doesn't return any errors.

**Comment by juliangruber** [comment]
Yes that's how it's working currently. I'm asking whether it would be nicer for you if it printed an error instead?

**Comment by ricarper** [comment]
> Yes that's how it's working currently. I'm asking whether it would be nicer for you if it printed an error instead?

Unfortunately it doesn't return any errors, it works normally but shows 0 rewards for a while.
